Source/CCZ4/Weyl4.impl.hpp

```diff
@@ -229,18 +226,16 @@ AMREX_GPU_DEVICE AMREX_FORCE_INLINE NPScalar_t<data_t> Weyl4::compute_Weyl4(
 // Defintions from gr-qc/0104063
 // "The Lazarus project: A pragmatic approach to binary black hole evolutions",
 // Baker et al.
-template <class data_t>
-AMREX_GPU_DEVICE AMREX_FORCE_INLINE Tetrad_t<data_t>
-Weyl4::compute_null_tetrad(const Vars<data_t> &vars,
-                           const Tensor<2, data_t> &h_UU,
-                           const Coordinates<data_t> &coords) const
+AMREX_GPU_DEVICE AMREX_FORCE_INLINE Tetrad_t Weyl4::compute_null_tetrad(
+    const Vars<amrex::Real> &vars, const Tensor<2, amrex::Real> &h_UU,
+    const Coordinates &coords) const
 {
-    Tetrad_t<data_t> out;
+    Tetrad_t out;
 
     // compute coords
-    const data_t x = coords.x;
-    const double y = coords.y;
-    const double z = coords.z;
+    const amrex::Real x = coords.x;
+    const double y      = coords.y;
+    const double z      = coords.z;
```

Comment:
Given you changed it in `Coordinates.hpp`:
```suggestion
    const amrex::Real x = coords.x;
    const amrex::Real y = coords.y;
    const amrex::Real z = coords.z;
```